TramsDataApi/UseCases/GetEstablishmentByUkprn.cs

```diff
@@ -16,6 +20,17 @@ public GetEstablishmentByUkprn(IEstablishmentGateway establishmentGateway)
         public EstablishmentResponse Execute(string ukprn)
         {
             var establishment = _establishmentGateway.GetByUkprn(ukprn);
+            return BuildResponse(establishment);
+        }
+
+        public EstablishmentResponse Execute(GetEstablishmentByUrnRequest request)
+        {
+            var establishment = _establishmentGateway.GetByUrn(request.URN);
+            return BuildResponse(establishment);
+        }
```

Comment:
I think this should live in it's own use case - keeping it in here makes it harder to refactor, and it's a little confusing for someone new to the project as they'll see the use case as being called `GetEstablishmentByUkprn`

Reply:
Yeah I can do that was just conscious that we just create duplication really. Would it make more sense to rename the use case to something like GetEstablishment rather than have two use cases that basically do the same thing?

Reply:
Yeah that works too 😄